Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12.8-alpine
```

Comment:
Consider using a specific patch version of the Python image to ensure consistent builds. For example, `python:3.12.8-alpine` might be replaced with `python:3.12.8-alpine3.17` to specify the Alpine version.